lib/xcode.js

```diff
@@ -267,6 +262,6 @@ function clearInternalCache () {
   });
 }
 
-export { getPath, getVersion, getAutomationTraceTemplatePath, getMaxIOSSDK,
+export default { getPath, getVersion, getAutomationTraceTemplatePath, getMaxIOSSDK,
```

Comment:
so we're exporting one object rather than exporting a number of functions.

this is where the difference in style helps denote a difference in meaning; since this is now just a regular JS object, I would take away the leading and trailing space inside the braces.